### ISSUE TITLE
Most recent 1.10.1 official container doesn't seem to work to pull on a Linux machine.

### ISSUE THREAD

**Issue by jtilander** [docker-library/golang#218]
This running on a Ubuntu 16.04 LTS:

```
Linux ct-host-u27 4.4.0-91-generic #114-Ubuntu SMP Tue Aug 8 11:56:56 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
```

```$ docker pull golang:1.10.0
1.10.0: Pulling from library/golang
c73ab1c6897b: Pull complete
1ab373b3deae: Pull complete
b542772b4177: Pull complete
57c8de432dbe: Pull complete
c81227e1ec90: Pull complete
3e5d5c68d1eb: Pull complete
a4ca93091ca5: Pull complete
Digest: sha256:ce70a7c27659bd714b63e7439ba0ae4a9f5a111fea16bd87575ddcfb0d0ef0cc
Status: Downloaded newer image for golang:1.10.0
$ docker pull golang:1.10
1.10: Pulling from library/golang
no supported platform found in manifest list```


New 1.10.1 image seems to have been pushed ~30m ago.

**Comment by TomWright** [comment]
I get a similar, although different error message:

```
$ docker pull golang:1.10.1
1.10.1: Pulling from library/golang
no matching manifest for linux/amd64 in the manifest list entries
```

```
Darwin 0260-MAC 16.7.0 Darwin Kernel Version 16.7.0: Mon Nov 13 21:56:25 PST 2017; root:xnu-3789.72.11~1/RELEASE_X86_64 x86_64
```

**Comment by jtilander** [comment]
Ah, ok. Closing this one then.

**Comment by jtilander** [comment]
Oh perhaps

**Comment by xZhad** [comment]
golang:stretch worked for me

still waiting for golang:latest

**Comment by tianon** [comment]
This is another instance of https://github.com/docker-library/official-images/issues/3835. :disappointed:

**Comment by sputnik13** [comment]
TL;DR for https://github.com/docker-library/official-images/issues/3835

hold your horses and try again later, it's eventual consistency

**Comment by tianon** [comment]
This should be fixed now.
